src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/Target_X64/X64ReadyToRunGenericHelperNode.cs

```diff
@@ -210,6 +210,7 @@ protected sealed override void EmitCode(NodeFactory factory, ref X64Emitter enco
                 case ReadyToRunHelperId.VirtualCall:
                 case ReadyToRunHelperId.ResolveVirtualFunction:
                 case ReadyToRunHelperId.MethodEntry:
+                case ReadyToRunHelperId.InterfaceDispatchCell:
```

Comment:
What's the difference between ResolveVirtualFunction and InterfaceDispatchCell? Shouldn't we have just one, and maybe name it `VirtualDispatchCell`? (There is nothing fundamental preventing the cell from being used to dispatch virtual methods. We are just not taking advantage of it today.)